packages/dds/tree/src/feature-libraries/chunked-forest/codec/format.ts

```diff
@@ -217,8 +218,17 @@ export type EncodedNodeShape = Static<typeof EncodedNodeShape>;
 export type EncodedAnyShape = Static<typeof EncodedAnyShape>;
 export type EncodedIncrementalChunkShape = Static<typeof EncodedIncrementalChunkShape>;
 
-export const EncodedFieldBatch = EncodedFieldBatchGeneric(
-	FieldBatchFormatVersion.v1,
+export const EncodedFieldBatchV1 = EncodedFieldBatchGeneric(
```

Comment:
Likely not worth it in this case, but one way we fond in the past to make the PRs more incremental and targeted for this kind of change is to do the rename to v1 in its own change, and setting up of the versioning as well. Thos can be done independent of adding the second version.